packages/components/data-table-manager/src/types.tsx

```diff
@@ -0,0 +1,139 @@
+import { type ReactNode, type MouseEventHandler, ReactElement } from 'react';
+
+import type { TDataTableSettingsProps } from './data-table-settings';
```

Comment:
I think we need to do this the other way around: define here the type and use it in the `DataTableSettings` component.
The way is currently implemented means the `DataTableManagerProvider` build package will include the `DataTableSettings` component because it's a transitive dependency.

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2807/commits/d1c167920c31da23e74275d1a3aa0912d539dc2d

Thank you